action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   publishDeployment:
     description: |
       Deployment ID on semgrep app.
+  generateSarif:
+    description: |
+      Set if you want a semgrep.sarif file to be written with all semgrep findings.
```

Comment:
Probably useful to say what to set it to... e.g. `1`, `true`, etc.

Reply:
Right. Any of them would work, but I'll say set to 1.